client/components/gpu-listings/GpuCard.js

```diff
@@ -0,0 +1,37 @@
+import style from './GpuCard.module.css'
+import Image from 'next/image'
+import HeartIcon from '../../public/icons/heart.svg'
+
+export default function GpuCard() {
+  return (
+    <div className={style.wrapper}>
+      <Image
+        priority
+        src="/images/asus-rog-gpu.jpg"
+        className={style.image}
+        height={200}
+        width={200}
+        alt="gpu thumbnail"
+      />
+      <hr className={style.divider} />
+      <div>
+        <span className={style.brand}>ASUS</span>
+        <h3 className={style.title}>ASUS ROG Strix NVIDIA GeForce RTX 3080 Ti OC Edition Gaming Graphics Card</h3>
+        <dl className={style.specs}>
```

Comment:
I can see the thinking behing using a `dl` here but semantically this should be a `ul` as you're using `li`'s

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

The note regarding "Permitted content"

Reply:
That makes sense, I will fix this.